src/data.json

```diff
@@ -8456,21 +8456,24 @@
     {
       "name": "print",
       "tags": [
-        "print"
+        "print",
+        "fax"
       ]
     },
     {
       "name": "print-outline",
       "tags": [
         "outline",
-        "print"
+        "print",
+        "fax"
       ]
     },
```

Comment:
it looks like the tags are always alphabetized in the list, even though there doesn't seem to be any rules actually checking that. perhaps we should alphabetize them here too?